src/System.Management.Automation/engine/hostifaces/PowerShell.cs

```diff
@@ -984,9 +986,11 @@ public PowerShell AddCommand(string cmdlet)
 
         /// <summary>
         /// Add a cmdlet to construct a command pipeline.
-        /// For example, to construct a command string "get-process | sort-object",
+        /// For example, to construct a command string "Get-Process | ForEach-Object { $_.Name }",
         ///     <code>
-        ///         PowerShell shell = PowerShell.Create("get-process").AddCommand("sort-object");
+        ///         PowerShell shell = PowerShell.Create().
+        ///                                     AddCommand("Get-Process").
+        ///                                     AddCommand("ForEach-Object { $_.Name }", true);
```

Comment:
`AddCommand` only takes the name of a command, so you cannot specify the argument in the string input.
For this overload, it's better use `PowerShell.Create().AddCommand("myscriptfunc.ps1", false)`, so the `myscriptfunc.ps1` will run in the calling scope, instead of a local scope.